Makefile

```diff
@@ -174,7 +174,7 @@ endif
 .PHONY: test-testsuite-do
 test-testsuite-do:
 	go vet ./...
-	go test -json=${JSON} -p 16 -parallel 4 -race -short=${SHORT} -timeout 10m -vet=off ./...
+	go test -json=${JSON} -cover -coverprofile=.build/coverprofile -p 16 -parallel 4 -race -short=${SHORT} -timeout 10m -vet=off ./...
```

Comment:
beware, this will overwrite the coverage of the main module

Reply:
Can this error be caused by this?
 `tparse error: no parseable events: call go test with -json flag`